### ISSUE TITLE
How to stop editor extensions?

### ISSUE THREAD

**Issue by mfreel** [raiguard/Factorio-EditorExtensions#28]
How do I stop the Mod?  I uninstall it but I still have infinite resources in my inventory and the time of day does not change.  How do I get the game to go back to normal?

**Comment by raiguard** [comment]
Re-enable the mod, go into the map editor, and un-freeze time in the time menu. Then type `/EditorExtensions disable-cheat-mode` into the console. After that you can remove the mod.

**Comment by mfreel** [comment]
Thanks for the quick reply.  Great mod, just can't get it to stop.  I have the time running.  When I run /EditorExtensions disable-cheat-mode when switching back to character (Cttl E, exit map editor) I get the error: 

Error while running command "EditorExtensions": Given value has to be larger than or equal to: 0
stack traceback:
	__EditorExtensions__/scripts/cheat-mode.lua:85: in function 'update_character_cheats'
	__EditorExtensions__/scripts/cheat-mode.lua:102: in function 'disable'
	__EditorExtensions__/control.lua:32: in function <__EditorExtensions__/control.lua:27>
stack traceback:
	[C]: in function '__newindex'
	__EditorExtensions__/scripts/cheat-mode.lua:85: in function 'update_character_cheats'
	__EditorExtensions__/scripts/cheat-mode.lua:102: in function 'disable'
	__EditorExtensions__/control.lua:32: in function <__EditorExtensions__/control.lua:27>

and it boots back to the start menu.  Disable the mod but still have infinite crafting that takes no time.

If I stay in the map editor and disable cheat mode  it says : disabled cheat mode, and so has disabled the testing tools for this force.  

Then it gives this error when Ctrl - E:

The mod Editor Extensions (1.6.0) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event EditorExtensions::ee-toggle-map-editor (ID 169)
The mod Editor Extensions (1.6.0) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event EditorExtensions::on_player_toggled_map_editor (ID 118)
Given value has to be larger than or equal to: 0
stack traceback:
	__EditorExtensions__/scripts/cheat-mode.lua:85: in function 'update_character_cheats'
	__EditorExtensions__/control.lua:449: in function <__EditorExtensions__/control.lua:406>
	[C]: in function 'toggle_map_editor'
	__EditorExtensions__/control.lua:328: in function <__EditorExtensions__/control.lua:325>
stack traceback:
	[C]: in function '__newindex'
	__EditorExtensions__/scripts/cheat-mode.lua:85: in function 'update_character_cheats'
	__EditorExtensions__/control.lua:449: in function <__EditorExtensions__/control.lua:406>
	[C]: in function 'toggle_map_editor'
	__EditorExtensions__/control.lua:328: in function <__EditorExtensions__/control.lua:325>
stack traceback:
	__EditorExtensions__/control.lua:328: in function <__EditorExtensions__/control.lua:325>
stack traceback:
	[C]: in function 'toggle_map_editor'
	__EditorExtensions__/control.lua:328: in function <__EditorExtensions__/control.lua:325>


Basically I don't have any crafting.  All the craft items are infinite and makes them instantly using no mats.  

I don't use any other mods.

Any thoughts?  Thanks for the help by the way.

**Comment by raiguard** [comment]
Oh boy, that's not good. I'm sorry! Could you please upload the save file so I can take a look?

**Comment by mfreel** [comment]
Thanks again for all the help and work you do.  Your mod is fantastic.  I'll keep an eye out for your patch.

**Comment by raiguard** [comment]
The patch has been released.

**Comment by raiguard** [comment]
Did that solve it for you?
